### DISCUSSION TITLE
Feature Request - find ALL identifications without an associated classification

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4880] [General]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
The [flat taxonomy gaps tool](https://arctos.database.museum/Reports/flat_taxonomy_gap.cfm) does not work for records using the A {string} formula. 

**Describe what you're trying to accomplish** 
Ensure that all of the identifications in a collection are associated with an appropriate classification

**Describe the solution you'd like**
It seems like we could easily do this by making the tool use just the "A" portion of any A {string} identification instead of the {string} which it now seems to do..

**Describe alternatives you've considered**
Just wait until I stumble upon these and add classifications.

**Additional context**
This affects any collection using an A {string} identification formula and means that if such an identification is associated with a taxon name that has no associated classification, there is no easy way to find and fix those.

**Priority**
Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair.
 Assigning high priority because I cannot check this for ANY identifications in UNM:MET see https://github.com/ArctosDB/data-migration/issues/501

**Comment by dustymc** [comment]
I understand the problem, but it has nothing to do with formulae - I'm not sure where that came from.

I don't think I can get there without some sort of a cache (https://github.com/ArctosDB/arctos/issues/3498), and that may or may not include "ALL" IDs - maybe depending on what (if anything) happens with https://github.com/ArctosDB/arctos/issues/3540.

**Comment by DerekSikes** [comment]
I was just exploring this tool. Nice!

I found one mystery: Bombus mckayi

In UAMObs:Ento this name comes back as lacking a classification -- but it
has a complete Arctos classification:

https://arctos.database.museum/name/Bombus%20mckayi

so why is this tool
https://arctos.database.museum/Reports/flat_taxonomy_gap.cfm

saying it lacks Kingdom, Phylum, Class, Order, Family, etc.? The specimens
are all ID formula A.

-Derek

On Tue, May 3, 2022 at 6:58 AM dustymc ***@***.***> wrote:

> I understand the problem, but it has nothing to do with formulae - I'm not
> sure where that came from.
>
> I don't think I can get there without some sort of a cache (#3498
> <https://github.com/ArctosDB/arctos/issues/3498>), and that may or may
> not include "ALL" IDs - maybe depending on what (if anything) happens with
> #3540 <https://github.com/ArctosDB/arctos/issues/3540>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/issues/4637#issuecomment-1116199383>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFNUM4PDYEH727IOEVZP23VIE5IRANCNFSM5U5IHNUQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

+++++++++++++++++++++++++++++++++++
*Derek S. Sikes*, Curator of Insects, Professor of Entomology
University of Alaska Museum (UAM), University of Alaska Fairbanks
1962 Yukon Drive, Fairbanks, AK   99775-6960
***@***.*** phone: 907-474-6278 he/him/his
University of Alaska Museum <https://www.uaf.edu/museum/collections/ento/>
-  search 357,704 digitized arthropod records
<http://arctos.database.museum/uam_ento>
+++++++++++++++++++++++++++++++++++

Interested in Alaskan Entomology? Join the Alaska Entomological
Society and / or sign up for the email listserv "Alaska Entomological
Network" at
http://www.akentsoc.org/contact_us

**Comment by dustymc** [comment]
>so why

Most excellent question - one record was missing the link between identification and taxonomy, I don't know how that could have happened - some sort of obscure bug in the taxonomy form, probably?? Anyway, it's fixed.

**Comment by DerekSikes** [comment]
Thanks!

On Tue, May 3, 2022 at 9:35 AM dustymc ***@***.***> wrote:

> so why
>
> Most excellent question - one record was missing the link between
> identification and taxonomy, I don't know how that could have happened -
> some sort of obscure bug in the taxonomy form, probably?? Anyway, it's
> fixed.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/issues/4637#issuecomment-1116366235>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFNUM4N3FUBDF4TJDWXC63VIFPVPANCNFSM5U5IHNUQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 

+++++++++++++++++++++++++++++++++++
*Derek S. Sikes*, Curator of Insects, Professor of Entomology
University of Alaska Museum (UAM), University of Alaska Fairbanks
1962 Yukon Drive, Fairbanks, AK   99775-6960
***@***.*** phone: 907-474-6278 he/him/his
University of Alaska Museum <https://www.uaf.edu/museum/collections/ento/>
-  search 357,704 digitized arthropod records
<http://arctos.database.museum/uam_ento>
+++++++++++++++++++++++++++++++++++

Interested in Alaskan Entomology? Join the Alaska Entomological
Society and / or sign up for the email listserv "Alaska Entomological
Network" at
http://www.akentsoc.org/contact_us

**Comment by Jegelewicz** [comment]
I was able to use the sql in the cheat sheet to get what I needed for now.
